inc/api/namespace.php

```diff
@@ -90,7 +90,7 @@ function get_package_data( WP_REST_Request $request ) {
 		);
 	}
 
-	wp_cache_set( 'fair-metadata-endpoint-' . $id, $response, 'metadata-endpoints', MiniFAIR\CACHE_LIFETIME );
+	set_transient( 'fair-metadata-endpoint-' . $id, $response, MiniFAIR\CACHE_LIFETIME );
```

Comment:
Where is this retrieved?

Reply:
It retrieved in fair-plugin when fair-plugin and mini-fair are both active for our issue with getting local package data.